nextpow2.py

```diff
@@ -0,0 +1,3 @@
+import numpy as np

```

Comment:
Could this fit in `denoise.py` instead? I think its a bit out of place to define 1 function in a single script.

Reply:
Sorry  I will solve it and delete the wav files